contracts/payloads/IGP86/PayloadIGP86.sol

```diff
@@ -214,6 +220,82 @@ contract PayloadIGP86 is PayloadIGPMain {
         ISmartLendingFactory(SMART_LENDING_FACTORY).setCreationCode(creationCode);
     }
 
+    // @notice Action 5: Transfer 500k FLUID to Team Multisig
+    function action5() internal isActionSkippable(5){
+        string[] memory targets = new string[](1);
+        bytes[] memory encodedSpells = new bytes[](1);
+
+        string
+            memory withdrawSignature = "withdraw(address,uint256,address,uint256,uint256)";
+
+        // Spell 1: Transfer INST to Team Multisig
+        {
+            uint256 FLUID_AMOUNT = 500_000 * 1e18; // 500k FLUID
+            targets[0] = "BASIC-A";
+            encodedSpells[0] = abi.encodeWithSignature(
+                withdrawSignature,
+                FLUID_ADDRESS,
+                FLUID_AMOUNT,
+                TEAM_MULTISIG,
+                0,
+                0
+            );
+        }
+
+        IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
+    }
+
+    // @notice Action 6: Delist mETH vaults
+    function action6() internal isActionSkippable(6){
+        {
+            address mETH_USDC_VAULT = getVaultAddress(81);
+            address mETH_USDT_VAULT = getVaultAddress(82); 
+            address mETH_GHO_VAULT = getVaultAddress(83);
+
+            // [TYPE 1] mETH<>USDC | collateral & debt
+            {
+                Vault memory VAULT_mETH_USDC = Vault({
```

Comment:
all of these we should set like we did the BOLD dex in last IGP: https://github.com/Instadapp/inst-governance/blob/main/contracts/payloads/IGP85/PayloadIGP85.sol#L169-L176

so just add `setBorrowProtocolLimitsPaused` helper method and then call that and `setBorrowProtocolLimitsPaused` for each vault here with the correct token

FYI: this is just to be as cautious as possible and go the full range of constricting that the configs allow. it's already paranoid to set limits when the vaults are paused anyway, but best security is being paranoid